### ISSUE TITLE
Exception thrown: "Please set module[name]=module_name instead of using script=path/to/script.php in your TCA."

### ISSUE THREAD

**Issue by spackmat** [alexanderschnitzler/templavoila#2]
| Q                    | A
| -------------------- | -----
| Bug report?          | yes
| Feature request?     | no
| PHP version          | 5.6
| TYPO3 version        | 7.6.18
| TemplaVoilà version  | 7.6.4

### Actual behaviour

The wizard for new link-elements sets `<script>browse_links.php?mode=wizard&act=file</script>` inside TCEForms->config->wizards->link. This is the old behaviour and leads to an exception, when one edits such a content element:

> The way registering a wizard in TCA has changed in 6.2 and was removed in CMS 7. Please set module[name]=module_name instead of using script=path/to/script.php in your TCA.

### Expected behaviour

According to the [Exception Info](https://wiki.typo3.org/Exception/CMS/1437750231), the following code snipped must be used. So instead of

````
<script>browse_links.php?mode=wizard&act=file</script>
````

use this:

````
<module type="array">
  <name>wizard_element_browser</name>
  <urlParameters type="array">
    <mode>wizard</mode>
    <act>url</act>
  </urlParameters>
</module>

````

This change should also be listed with the other breaking changes, as all old datastructures containing link fields must be updated manually.

### Steps to reproduce the behaviour

- I have datastructures containing fields of type link (or I create a new one)
- I create new content with this element
- I get the mentiones exception

**Comment by alexanderschnitzler** [comment]
Thanks for the report. For now, please fix this manually in your created data structures. I will work on 7.6.5 these days which will contain a fix for this issue. Don't expect a release this week. There are some more minor issues, that will be fixed with 7.6.5 as well.

**Comment by spackmat** [comment]
Nice! Don't forget to mention this code change in the documentation a a breaking change. It took me a while to find out, what exactly triggered this Exception in my migrated FCEs.
